governance-charter.md

```diff
@@ -112,19 +122,17 @@ to the respective positions.
 
 ### Maximal representation
 
-To encourage diversity there will be a maximum of one-third representation on
-the Governance Committee from any one company at any time. If the outcomes of
-an election result in greater than 1/3 representation (or maximum of two,
-whichever is greater), the lowest vote getters from any particular company will
-be removed until representation on the committee is less than one-third.
-
-If percentages shift because of job changes, acquisitions, or other events,
-sufficient members of the committee must resign until max one-third
-representation is achieved. If it is impossible to find sufficient members to
-resign, the entire company’s representation will be removed and new special
-elections held. In the event of a question of company membership (for example
-evaluating independence of corporate subsidiaries) a majority of all
-non-involved Governance Committee members will decide. 
+To encourage diversity there will be a maximum of two Governance Committee
+members from any one company at any time.
```

Comment:
```suggestion
To encourage diversity there will be a maximum of two Governance Committee
members from any one company at any time. Therefore, there are limits on the eligibility for employees from the same company (see [Eligibility for voting](#eligibility-for-voting)). 

If the outcomes of an election would result in more than two GC members from the same company the following rules will be applied to ensure maximal representation: In the case of one existing GC member of that company, who was not up for re-election, this existing member remains, and all except the highest vote getter from any particular company will be removed. In the case of no existing GC member of that company, all except the two highest vote getters from any particular company will be removed. 
```

Reply:
If people are not eligible when there are already two colleagues on the board, when would this outcome happen?

Reply:
If there are already 2 GC members of a company, yes, there is no problem.

But, if there is only 1 GC member of a company, and additional `n` employees of the same company run, there could be an outcome where there is 1 GC member and `n` new. Only one of `n` can get a seat then.

And even if there is 0 GC member of a company, there is still the possibility that `n` get elected and the top 2 of them get a seat then.

Reply:
It's probably common sense that the most voted will be the one getting a seat from the number of available seats per company (which could be 1 or 2), but I think it should be stipulated here in some way to avoid any doubt.

Reply:
Fair point, you could infer that from the rest of the election rules laid out, but being explicit might help to avoid future issues

Reply:
solved by the https://github.com/open-telemetry/community/pull/1764/commits/73b3e9d0ead0edb39567f8e46b62fe1d9bd99428